test/spark/run-client-test.sh

```diff
@@ -1,44 +1,121 @@
 #!/bin/bash -aux
 
-# Run Export
-docker-compose run -v  ${CLIENT_JAR}:/client/client.jar -T --no-deps --rm spark-submit bash -c 'spark-submit  --master spark://spark:7077 --conf spark.hadoop.lakefs.api.url=http:/docker.lakefs.io:8000/api/v1   --conf spark.hadoop.lakefs.api.access_key=${TESTER_ACCESS_KEY_ID}   --conf spark.hadoop.fs.s3a.connection.ssl.enabled=false   --conf spark.hadoop.lakefs.api.secret_key=${TESTER_SECRET_ACCESS_KEY}   --class io.treeverse.clients.Main  /client/client.jar test-data ${EXPORT_LOCATION}   --branch=main'
+run_lakectl() {
+  docker-compose exec -T lakefs lakectl "$@"
+}
 
-# Validate export
-lakectl_out=$(mktemp)
-s3_out=$(mktemp)
-trap 'rm -f -- $s3_out $lakectl_out' INT TERM EXIT
+run_gc () {
+  docker-compose run  -v  ${CLIENT_JAR}:/client/client.jar -T --no-deps --rm spark-submit bash -c 'spark-submit --master spark://spark:7077 --class io.treeverse.clients.GarbageCollector   -c spark.hadoop.lakefs.api.url=http://docker.lakefs.io:8000/api/v1 -c spark.hadoop.lakefs.api.access_key=${TESTER_ACCESS_KEY_ID}    -c spark.hadoop.lakefs.api.secret_key=${TESTER_SECRET_ACCESS_KEY}     -c spark.hadoop.fs.s3a.access.key=${LAKEFS_BLOCKSTORE_S3_CREDENTIALS_ACCESS_KEY_ID}   -c spark.hadoop.fs.s3a.secret.key=${LAKEFS_BLOCKSTORE_S3_CREDENTIALS_SECRET_ACCESS_KEY} /client/client.jar $1 us-east-1'
+}
 
-docker-compose exec -T lakefs lakectl fs ls --recursive --no-color lakefs://test-data/main/ | awk '{print $8}' | sort > ${lakectl_out}
+clean_repo() {
+  local repo=$1
+  branch_names=$(run_lakectl branch list lakefs://${repo} | awk '{print $1}')
```

Comment:
Not sure why `repo` is local but `branch_names`, `branch` and others are not.

Reply:
You're right.
Fixed